validation/validationFunction/ww_validation_prototype.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+pip install cerberus
+
+#importing necessary packages
+
+from cerberus import Validator
+import pandas as pd
+import yaml
+
+#load the csv file
+from google.colab import files
+uploaded = files.upload()
+
+#%%
```

Comment:
Not sure what the script does - there is a lot of commented out code and no docstrings to explain what the goal of the script is, or what any of the fuunctions do.
Plus, the script is not called in a main() function, which is not ideal 

Since it's a prototype, should this script be included in the pull request?